src/classes/RD2_StatusMapper.cls

```diff
@@ -169,6 +169,18 @@ public with sharing class RD2_StatusMapper {
         return mapping == null ? null : mapping.state;
     }
 
+    public Set<String> getAllClosedStatus() {
```

Comment:
Let's add a description for this method, and maybe rename it? `getClosedStatusValues`?